### PR TITLE
SCM - Source Control Repositories view to use scm.providerCountBadge

### PR DESCRIPTION
**Pull request by lszomoru**

### CODE REVIEW

File: src/vs/workbench/contrib/scm/browser/scm.contribution.ts [hunk 1]
```diff
@@ -227,7 +227,7 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration).regis
 				localize('scm.providerCountBadge.auto', "Only show count badge for Source Control Provider when non-zero."),
 				localize('scm.providerCountBadge.visible', "Show Source Control Provider count badges.")
 			],
-			description: localize('scm.providerCountBadge', "Controls the count badges on Source Control Provider headers. These headers only appear when there is more than one provider."),
+			markdownDescription: localize('scm.providerCountBadge', "Controls the count badges on Source Control Provider headers. These headers appear in the \"Source Control\", and \"Source Control Sync\" views when there is more than one provider or when the `#scm.alwaysShowRepositories#` setting is enabled, as well as in the \"Source Control Repositories\" view."),
```
Comment (Tyriar) [inline src/vs/workbench/contrib/scm/browser/scm.contribution.ts:230]: You should move the `#scm.alwaysShowRepositories#` part into a variable so translators don't break the link or setting name.

**Review by Tyriar — CHANGES_REQUESTED** [review]

**Comment by bilogic** [comment]
@lszomoru

Does this mean that repos with `0` changes will no longer have a badge if I set this to `auto`?

**Comment by lszomoru** [comment]
@bilogic, that is correct. I have found out that we already have a setting and established behaviour that is driven by this setting in the "Source Control" view. Since the "renderer" that renders that repository header is the same in both views ("Source Control" and "Source Control Repositories") it felt natural to adopt this setting.

**Comment by bilogic** [comment]
@lszomoru thanks, that sounds similar to `-`, in that we do just enough to deal with the main issue and nothing more lest it creates other problems. If we can quickly move this to production, I can give real world feedback. Thank you!

File: src/vs/workbench/contrib/scm/browser/scm.contribution.ts (hunk 1 above)
Comment (bilogic) [reply src/vs/workbench/contrib/scm/browser/scm.contribution.ts:230]: @lszomoru This is the change required, how can I commit it in?

```
markdownDescription: localize('scm.providerCountBadge', "Controls the count badges on Source Control Provider headers. These headers appear in the \"Source Control\", and \"Source Control Sync\" views when there is more than one provider or when the {0} setting is enabled, as well as in the \"Source Control Repositories\" view.", `\`#scm.alwaysShowRepositories#\``),
```

Comment (bilogic) [reply src/vs/workbench/contrib/scm/browser/scm.contribution.ts:230]: https://github.com/microsoft/vscode/pull/194967

Comment (bilogic) [reply src/vs/workbench/contrib/scm/browser/scm.contribution.ts:230]: @lszomoru is my change ok?

Comment (bilogic) [reply src/vs/workbench/contrib/scm/browser/scm.contribution.ts:230]: I can't test this until it is in insider or production

**Review by jrieken — DISMISSED** [review]

File: src/vs/workbench/contrib/scm/browser/scm.contribution.ts [hunk 2]
```diff
@@ -227,7 +227,7 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration).regis
 				localize('scm.providerCountBadge.auto', "Only show count badge for Source Control Provider when non-zero."),
 				localize('scm.providerCountBadge.visible', "Show Source Control Provider count badges.")
 			],
-			description: localize('scm.providerCountBadge', "Controls the count badges on Source Control Provider headers. These headers only appear when there is more than one provider."),
+			markdownDescription: localize('scm.providerCountBadge', "Controls the count badges on Source Control Provider headers. These headers appear in the \"Source Control\", and \"Source Control Sync\" views when there is more than one provider or when the `{0}` setting is enabled, as well as in the \"Source Control Repositories\" view.", '#scm.alwaysShowRepositories#'),
```
Comment (Tyriar) [inline src/vs/workbench/contrib/scm/browser/scm.contribution.ts:230]: Ideally the backticks would also be in the non localized part as they're critical to formatting:

```suggestion
			markdownDescription: localize('scm.providerCountBadge', "Controls the count badges on Source Control Provider headers. These headers appear in the \"Source Control\", and \"Source Control Sync\" views when there is more than one provider or when the {0} setting is enabled, as well as in the \"Source Control Repositories\" view.", '\`#scm.alwaysShowRepositories#\`'),
```

**Review by Tyriar — CHANGES_REQUESTED** [review]

**Review by Tyriar — APPROVED** [review]

**Pull request merged**